README.md

```diff
@@ -188,6 +188,44 @@ We can always use layer to search for data availability:
   const datesS1 = await layerS1.findDatesUTC(bbox, fromTime, toTime);
 ```
 
+
+## Cancelling requests
+
+You can also cancel requests when searching/fetching data.
+
+To do so a factory of tokens needs to be created and a token passed through a request configuration object. In this object can also be defined the amount of retires desired in the case the petition fails.
+
+In the example below, a cancelFactory is created, passing a token inside the configuration request object. The timeout will cancel the requests after 500 miliseconds, throwing an exception.
+
+This exception can be caught and identified by isCancelled, function that can be imported from the library.
```

Comment:
- isCancelled => `isCancelled` (backticks should be added)

- I would suggest leaving out that `isCancelled` can be imported from the library (it is true for everything else too).